### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by dumboluzz**
Perfect PR to review on a monday morning. 😄

### CODE REVIEW

**Review by finnstue — COMMENTED** [review]
Good work

File: app/views/scans/new.html.erb [hunk 1]
```diff
@@ -25,15 +27,21 @@ function showPosition(position) {
   fetch("/shipments/" + id.dataset.shipment + "/scans", {
     method: "POST",
     headers: {
-    "X-CSRF-Token": csrfToken, // Set the token
-    "Content-Type": "application/json"
+      "X-CSRF-Token": csrfToken, // Set the token
+      "Content-Type": "application/json",
+      "Accept": "application/json"
```
Comment (finnstue) [inline app/views/scans/new.html.erb:32]: What does this do?

Comment (finnstue) [reply app/views/scans/new.html.erb:32]: The "Accept"

Comment (dumboluzz) [reply app/views/scans/new.html.erb:32]: This means "As a response I only accept json". In the controller I check the requested format with `respond_to`.

**Review by finnstue — APPROVED** [review]

**Review by sophiecs333 — APPROVED** [review]

**Pull request merged**